solvebio/cli/data.py

```diff
@@ -942,7 +942,9 @@ def ls(args):
               "Try the --recursive flag instead.")
         return False
 
-    files = _ls(args.full_path, recursive=args.recursive)
+    follow_shortcuts = True if args.follow_shortcuts else False
```

Comment:
why do we need this if we can just use `args.follow_shortcuts`

Reply:
Addressed these comments a while ago - forgot to follow up here in the thread sorry